Commands/escaperooms.py

```diff
@@ -0,0 +1,548 @@
+                    escapeusernames = usernames_from_ids(ctx, escapeusers)
+                    message += translate("esc_room_today_players", cult).format(
+                        ", ".join(escapeusernames),
+                        white_check_mark
+                    )
+                else:
+                    message += translate("esc_room_today_nolayers",
+                                         cult).format(white_check_mark)
+
+                timeout = time.time() + TIMEOUT
+                msg = await ctx.send(message)
+                await msg.add_reaction(white_check_mark)
+                reaction_added_task = asyncio.create_task(
+                    self.wait_for_added_reactions(
+                        ctx, msg, guild_data, timeout)
+                )
+                reaction_removed_task = asyncio.create_task(
+                    self.wait_for_removed_reactions(
+                        ctx, msg, guild_data, timeout)
+                )
+
+                await reaction_added_task
+                await reaction_removed_task
+                await msg.delete()
+
+            else:
+                await guild_data.clear_escaperoom_users()
+                await ctx.send(translate("esc_not_scheduled", cult))
+        else:
+            await ctx.send(translate("esc_not_scheduled", cult))
+
+    def calculate_room(self, user_list) -> str:
+        """ Calculates the best room available for the users in list"""
+        tracking_table = self.read_tracking_table()
+
+        # calculate sum of each room for full userbase
+        room_sums = pd.DataFrame(tracking_table.drop(
+            columns=['Naam', 'Discord ID', 'Heeft tabletop']).sum(axis=0, skipna=True))
+        room_sums.columns = ['sum_total']
+
+        # Filter table on current users
+        subset = tracking_table.loc[tracking_table['Discord ID'].isin(
+            user_list)]
+
+        # calculate sum of each room for current users
+        sub_room_sums = pd.DataFrame(subset.drop(
+            columns=['Naam', 'Discord ID', 'Heeft tabletop']).sum(axis=0, skipna=True))
+        sub_room_sums.columns = ['sum']
+
+        # keep only rooms not yet played by current users
+        sub_room_sums = sub_room_sums[sub_room_sums['sum'] < 1]
+
+        # join in the number of plays by the full userbase
+        sub_room_sums = sub_room_sums.join(room_sums)
+
+        # sort descending
+        sub_room_sums.sort_values(
+            by=['sum_total'], ascending=False, inplace=True)
+        sub_room_sums.reset_index(level=0, inplace=True)
+
+        # pick first room
+        return sub_room_sums.loc[0][0]
+
+    def split_group(self, user_list, is_balanced=False) -> list:
+        tracking_table = self.read_tracking_table()
+        user_list = list(map(str, user_list))
+
+        # Clean table and transpose
+        subset = tracking_table.loc[tracking_table['Discord ID'].isin(
+            user_list)]
+        debug(user_list)
```

Comment:
you might want to add some extra info here.
This is great while debugging the specific feature, but when looking back into the logs, this will look like some random data :)

Reply:
To be honest, we could also remove the debugs here. was good for testing.